### PR TITLE
v2.0.10 bugfixes

### PR DESCRIPTION
**Pull request by bastille**
Bugfixes to address newly added data that is not properly processed by Ariel

## Description
Added 2 filters to remove incompatible data.  Added one option to force processing to deal with incompatible data.

## Motivation and Context
Ariel stopped working for with our CUR data

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

## License  
I confirm that this contribution is made under the Apache License, Version 2.0 and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: ariel/generate_reports.py [hunk 1]
```diff
@@ -500,7 +500,7 @@ def get_units(instancetype):
 
     # GroupBy to assign appropriate index columns
     unused_az_ris = unused_az_ris.groupby(az_instance_groups.keys).sum()
-    ri_hourly_usage_report = ri_hourly_usage_report.groupby(region_instance_groups.keys + ['hourofweek']).sum()
+    ri_hourly_usage_report = ri_hourly_usage_report.groupby(region_instance_groups.keys + ['hourofweek']).sum(numeric_only=None)
```
Comment (meckstmd) [inline ariel/generate_reports.py:503]: I am not incredibly well versed in Pandas but it looks to me from the docs that `numeric_only=None` is the default so this isn't needed (although this doesn't cause any issue either).

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.sum.html

**Review by meckstmd — APPROVED** [review]
👍

File: ariel/generate_reports.py (hunk 1 above)
Comment (bastille) [reply ariel/generate_reports.py:503]: Very close.  This is applying to a GroupBy, not a DataFrame, so the doc is https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.core.groupby.GroupBy.sum.html, which defaults to True.

I'm not sure why I needed to add this change.  When I looked at the source data I did not find any non-numeric data in the input.  My only guess was that it was reading some value like 0.13833299999999998 as non-numeric, which shouldn't be the case, but adding this flag fixed one of the many bugs I ran into earlier this week.

Comment (meckstmd) [reply ariel/generate_reports.py:503]: Ok, cool - thanks for the clarification.

**Pull request merged**